Test/selenium_test.py

```diff
@@ -20,10 +24,20 @@ def driver():
     with webdriver.Chrome(ChromeDriverManager().install(), options=opts) as driver:
         yield driver
 
+
 @pytest.fixture(scope='module')
 def test_app():
+    # when flask isnt running, selenium test runs as readonly. but when session is running it asks to authenticate?
+    ##added this to pick up LOGIN_DISABLED flag from flask_config to turn off authenticatin for testing
+    os.environ['LOGIN_DISABLED'] = 'True'
+    #os.environ['ANON_USER'] = 'TheLegendaryPan'## added as part of module 10
```

Comment:
With this commented out, the test fails when run locally (unless the user manually sets this environment variable)